sdk/servicebus/Microsoft.Azure.ServiceBus/changelog.md

```diff
@@ -0,0 +1,25 @@
+# 4.0.0
+## Breaking Change
```

Comment:
nit:  This should probably be "Breaking Changes" since it's multiple items in a list.